### PR TITLE
Add VPC Flow Logs for Network Security

### PR DESCRIPTION
**Pull request by bbayles**
This PR adds a note about monitoring VPC activity with VPC Flow Logs. It links to the official documentation, to an open source command line tool / library for reading the data, and to services that can make use of the data.

__Disclosure__: The CLI tool is sponsored by my employer, [Observable Networks](http://observable.net/), which operates a network security monitoring service that uses VPC Flow Logs.

### CODE REVIEW

File: AUTHORS.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ The following people (in alphabetical order) have contributed to or reviewed thi
 * [Alexander Atallah (alexanderatallah)](https://github.com/alexanderatallah)
 * [Artem Nikitin (artemnikitin)](https://github.com/artemnikitin) — [2+](https://github.com/open-guides/og-aws/commits?author=artemnikitin)/[1+](https://github.com/open-guides/og-aws/issues?q=author%3Aartemnikitin)
 * [Ben Kehoe (benkehoe)](https://github.com/benkehoe) — [4+](https://github.com/open-guides/og-aws/commits?author=benkehoe)/[1+](https://github.com/open-guides/og-aws/issues?q=author%3Abenkehoe)
+* [Bo Bayles (bbayles)](https://github.com/bbayles)
```
Comment (ThanosBaskous) [inline AUTHORS.md]: This file is periodically regenerated by the project leads (@jlevy and myself) - no need to edit it yourself. Would you mind removing this change?

File: README.md [hunk 2]
```diff
@@ -1106,6 +1106,10 @@ VPCs, Network Security, and Security Groups
 	-	You expose a smaller surface area for attack compared to exposing separate (potentially authenticated) services over the public internet.
 		-	e.g. A bug in the YAML parser used by the Ruby on Rails admin site is much less serious when the admin site is only visible to the private network and accessed through VPN.
 	-	Another common pattern (especially as deployments get larger, security or regulatory requirements get more stringent, or team sizes increase) is to provide a [bastion host](https://www.pandastrike.com/posts/20141113-bastion-hosts) behind a VPN through which all SSH connections need to transit.
+-   **VPC Flow Logs** allow you to monitor the network traffic to, from, and within your VPC. Logs are stored in CloudWatch Logs groups, and can be used for security monitoring (with 3rd party tools), performance evaluation, and forensic investigation.
```
Comment (ThanosBaskous) [inline README.md]: Consistency nit: we prefer "third-party" to "3rd party". I just noticed that one "3rd" crept into master and will be fixing it.

File: README.md [hunk 3]
```diff
@@ -1106,6 +1106,10 @@ VPCs, Network Security, and Security Groups
...
+    -	See the [VPC Flow Logs User Guide](http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/flow-logs.html) for basic information.
+    -	See the [flowlogs-reader](https://github.com/obsrvbl/flowlogs-reader) CLI tool and Python library to retrieve and work with VPC Flow Logs.
+    -	Services like [Observable Networks](https://observable.net/) and [Sumo Logic](https://www.sumologic.com/) can use VPC Flow Logs as input for security monitoring and/or searching.
```
Comment (ThanosBaskous) [inline README.md]: Heads up: we are currently figuring out how to handle addition of third-party vendors (as the guide has gotten popular, lots of people have attempted to add companies - nothing inherently wrong with that, but it's probably a sustainable approach to add all of them to guide). This PR may sit for a bit while we figure it out - if you remove this line (and resubmit as a separate PR, if you'd like), I'd be happy to push this one through to the next review stage.

File: AUTHORS.md (hunk 1 above)
Comment (bbayles) [inline AUTHORS.md]: Whoops, got it. I've reverted the change.

File: README.md (hunk 2 above)
Comment (bbayles) [inline README.md]: OK, fixed.

**Comment by bbayles** [comment]
I've made the requested changes for this one - thanks for reviewing (and for making this guide!).

**Comment by ThanosBaskous** [comment]
Thanks for the contribution and fixes @bbayles!

@richadams would you mind doing a content review of this PR?

**Comment by richadams** [comment]
Content looks good to me! 👍

**Comment by bbayles** [comment]
I've pulled in in master to fix the merge conflict. Thanks for approving; I'll be sure to add more once this is merged!

**Comment by ThanosBaskous** [comment]
Thanks all!

**Pull request merged**